packages/algob/src/internal/deployer.ts

```diff
@@ -246,6 +247,21 @@ export class DeployerDeployMode implements AlgobDeployer {
   async loadLsig (name: string, scParams: Object): Promise<Object> {
     return await getLsig(name, scParams, this.algoOp.algodClient);
   }
+
+  /**
+   * Description : loads multisigned logic signature from .mlsig file
+   * @param {string} name filename
+   * @param {Object} scParams parameters
+   * @returns {Object} multi signed logic signature from assets/<file_name>.mlsig
+   */
+  async loadMultiSig (name: string, scParams: Object): Promise<Object> {
+    const [tealFile, Msig]: any = await readMsigFromFile(name); // Get tealFile name, Msig object from .mlsig
+    const lsig: any = await this.loadLsig(tealFile, scParams); // Load lsig from .teal (getting logic part from lsig)
```

Comment:
Don't use `any`. `lsig` should have a proper, well defined type. Same for line 258: tealFile and Msig

Reply:
created `LogicSig` interface for logic signature in `types.ts`.